app/src/test/java/woowacourse/movie/domain/discountPolicy/OffTimeTest.kt

```diff
@@ -0,0 +1,54 @@
+package woowacourse.movie.domain.discountPolicy
+
+import org.junit.Assert.assertEquals
+import org.junit.Test
+import woowacourse.movie.domain.Price
+import woowacourse.movie.domain.ReservationDetail
+import java.time.LocalDateTime
+
+class OffTimeTest {
+    @Test
+    fun `11시 이전일 경우 2000원 할인이 적용된다`() {
+        // given
+        val date = LocalDateTime.of(2023, 1, 10, 10, 0)
```

Comment:
```suggestion
        val date = LocalDateTime.of(2023, 1, 10, 10, 59)
```
가급적이면 경계값을 위주로 테스트를 작성하시는 것이 좋습니다. (이하 관련 내용 동일)